integrated_channels/blackboard/migrations/0013_alter_blackboardlearnerdatatransmissionaudit_index_together.py

```diff
@@ -19,23 +18,44 @@ class Migration(migrations.Migration):
             ),
         ]
     else:
-        operations = [
-            migrations.SeparateDatabaseAndState(
-                state_operations=[
-                    migrations.AlterIndexTogether(
-                        name='blackboardlearnerdatatransmissionaudit',
-                        index_together={('enterprise_customer_uuid', 'plugin_configuration_id')},
-                    ),
-                ],
-                database_operations=[
-                    migrations.RunSQL(sql="""
-                        CREATE INDEX blackboard_bldta_85936b55_idx
-                        ON blackboard_blackboardlearnerdatatransmissionaudit (enterprise_customer_uuid, plugin_configuration_id)
-                        ALGORITHM=INPLACE LOCK=NONE
-                    """, reverse_sql="""
-                        DROP INDEX blackboard_bldta_85936b55_idx
-                        ON blackboard_blackboardlearnerdatatransmissionaudit
-                    """),
-                ]
-            ),
-        ]
+        if 'postgresql' in db_engine:
```

Comment:
Nit: I think this would read a bit easier if it was in an "elif" so that each set of operations is at the same indentation level, i.e.:

```python
if 'sqlite3' in db_engine:
    # ...
elif 'postgresql' in db_engine:
    # ....
else:
    # ...
```

Reply:
I would agree it make sense.